src/containers/Home/HomeWidgets/NotificationWidget.js

```diff
@@ -0,0 +1,278 @@
+            }}>
+            <Card
+                style={{
+                    width: width - 20,
+                    borderRadius: 10,
+                    backgroundColor: '#F1F1F1'
+                }}
+                mode="elevated"
+                elevation={5}>
+                <Card.Content>
+                    <View
+                        style={{
+                            display: 'flex',
+                            flexDirection: 'row',
+                            justifyContent: 'space-between',
+                            width: width - 30,
+                            height: 10,
+                            alignItems: 'center',
+                            marginBottom: collapsed ? 0 : 15,
+                        }}>
+                        <View style={{
+                            display: 'flex',
+                            flexDirection: 'row',
+                            alignItems: 'center',
+                        }}>
+                            <View
+                                style={{
+                                    display: 'flex',
+                                    flexDirection: 'row',
+                                    alignItems: 'center',
+                                    backgroundColor: '#56B73E',  // Set the background color to green
+                                    borderRadius: 5,  // Set the border radius for rounded corners
+                                    height: 24,  // Set the height
+                                    width: 55,  // Set the width
+                                    marginRight: 2
+                                }}>
+                                <IconButton
+                                    icon="bell"
+                                    color="white"
+                                    size={15}
+                                    style={{ marginRight: 3 }}
+                                />
+                                <View style={{ justifyContent: 'center', alignItems: 'center', borderColor: 'white', borderWidth: 1, borderRadius: 15, width: 16, height: 16 }}>
+                                    <Text style={{ color: "white", textAlign: 'center', fontSize: 10 }}>
+                                        {traynotifications.length}
+                                    </Text>
+                                </View>
+                            </View>
+                            {collapsed && traynotifications.map((notification) =>
+                                notification.icon)}
+                        </View>
+                        <View style={{
+                            display: 'flex',
+                            flexDirection: 'row',
+
+                        }}>
+                            <TouchableOpacity
+                                onPress={() => {
+                                    Alert.alert("Clear Notifications", "Are you sure you want to clear all notifications?",
```

Comment:
If a notification for a pending VerusID is removed, the pending VerusID seems to stay in async storage forever, with no way to remove it. It would be ideal to either sweep pending VerusIDs on app start based on age (maybe older than a week or so) and then, when the notification is pressed, tell the user that the pending ID has expired if they try to link it (or automatically remove the notification, if it exists, once the pending ID is removed), or remove the pending ID when the notification is cleared. Another possible solution is to remove pending IDs that are not tied to any notification.